### ISSUE TITLE
Intellisense not working in WSL

### ISSUE THREAD

**Issue by marko1010** [microsoft/vscode#231159]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.94.2
- OS Version: Windows 11 (WSL)

Steps to Reproduce:

1. Open WSL workspace in VS Code
2. Install node & npm using nvm
3. Create an index.ts file and import anything

Intellisense will not work for any of the imports. On Windows (outside of WSL) it works fine.

WSL:
![Image](https://github.com/user-attachments/assets/41d0867f-6eb8-4726-a4a7-a6c83cad94d1)

Windows:
![Image](https://github.com/user-attachments/assets/9777fa85-9263-4cae-9f81-25a70a3acaf4)

It also doesn't work when developing via Remote SSH extension.

It could be something on my end but I spent countless hours trying to fix this to no avail.

It starts TS server fine (screenshot of TypeScript output window), but it uses tsserver from the host and not WSL. Could that be a problem?

![Image](https://github.com/user-attachments/assets/9c6f2635-0831-4a6f-a7f1-a14c3f66712b)

I try to force it to use WSL's tsserver by specifying path in workspace json settings: 
"typescript.tsdk": "/home/marko/.nvm/versions/node/v20.18.0/lib/node_modules/typescript/lib"

But I get the following error despite the fact that the file exists at the given path:

![Image](https://github.com/user-attachments/assets/1be7b8a8-2a9d-486f-89e0-7c31d0b23250)

Notice how the error message shows backslashes instead of forward slashes in the file path

**Comment by cesarvspr** [comment]
Same issue here. No idea how to fix.

**Comment by mjbvz** [comment]
Are you using the [wsl remote extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl) to open the workspace or directly opening the wsl folder through windows? It looks it could be the second case, which you almost almost never want. In the true wsl case, the typescript server will automatically run under the wsl linux environment too. 

Can you try using the wsl extension to open the workspace instead?

**Comment by cesarvspr** [comment]
![Image](https://github.com/user-attachments/assets/45fb5cc6-d7e0-4098-99a0-8f956288d31b)
![Image](https://github.com/user-attachments/assets/19274516-d2ce-4333-b0f1-e68e84c46ed4)

So many things puzzle me. I see "initializing project... " and it works on startup but stops later. And the node VIRT memory usage seems outa of hand

**Comment by marko1010** [comment]
> Are you using the [wsl remote extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl) to open the workspace or directly opening the wsl folder through windows? It looks it could be the second case, which you almost almost never want. In the true wsl case, the typescript server will automatically run under the wsl linux environment too.
> 
> Can you try using the wsl extension to open the workspace instead?

Yes, I'm using WSL extension to open the workspace and TS server seems to start but I still get no IntelliSense

![Image](https://github.com/user-attachments/assets/a073ab41-31a2-43a9-a788-08ca8707ed0b)
